python/README.md

```diff
@@ -0,0 +1,94 @@
+# The Cucumber Compatibility Kit for Python
+
+The CCK (Cucumber Compatibility Kit) is a set of features and messages. It aims to validate an implementation of the [Cucumber Messages protocol](https://github.com/cucumber/common/tree/main/messages#cucumber-messages).
+
+## Overview
+
+This kit (package) consists of a set of features, miscellaneous files, and messages:
+
+- Each area will contain one feature, which, once executed, will emit an exhaustive set of messages as specified by the protocol.
+- Some of these areas may "also" require miscellaneous files to be used when testing functions such as attaching images or documents or reading data from files.
+- Each area will contain a set of messages - serialised as a single `.ndjson` file.
+
+This is the reference for the CCK: a given feature from the kit, when executed using any dedicated step definitions, must emit the **exact** corresponding messages.
+
+## Installation and Usage
+
+1. Install the package using `pip`. You can do this by adding `cucumber-compatibility-kit` to your `requirements.txt` or using `pip` directly:
+
+    ```bash
+    pip install cucumber-compatibility-kit
+    ```
+
+2. Set up your test environment. Here's an example of how you might set up a test using `pytest`:
+
+    ```python
```

Comment:
Important TOFIX: This example / documentation should be added to be something functionally equivalent to "HOW" your cck will run in the consuming cucumber flavour.

The sample here is the internal tests for the library which wouldn't be copied by a consuming cucumber. For cucumber ruby the documentation example is here: https://github.com/cucumber/compatibility-kit/blob/main/ruby/README.md#installation-and-usage